x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/analytics_list/expanded_row.tsx

```diff
@@ -18,7 +18,13 @@ import { DataFrameAnalyticsListRow } from './common';
 import { ExpandedRowDetailsPane, SectionConfig } from './expanded_row_details_pane';
 import { ExpandedRowJsonPane } from './expanded_row_json_pane';
 import { ProgressBar } from './progress_bar';
-import { getDependentVar, getValuesFromResponse, loadEvalData, Eval } from '../../../../common';
+import {
+  getDependentVar,
+  getPredictionFieldName,
+  getValuesFromResponse,
+  loadEvalData,
+  Eval,
+} from '../../../../common';
 import { isCompletedAnalyticsJob } from './common';
 import { isRegressionAnalysis } from '../../../../common/analytics';
 // import { ExpandedRowMessagesPane } from './expanded_row_messages_pane';
```

Comment:
commented import (not part of the changes, but should be removed I guess)

Reply:
I believe this was added in the original code for when the messages become available - which I think they are now if not very soon. I think it's worth keeping in as it will be used very soon. 😄